bin/s2nc.c

```diff
@@ -79,6 +80,9 @@ void usage()
     fprintf(stderr, "    Turn on corked io\n");
     fprintf(stderr, "  --tls13\n");
     fprintf(stderr, "    Turn on experimental TLS1.3 support.\n");
+    fprintf(stderr, "  -k ,--keyshares\n");
+    fprintf(stderr, "    Colon separated list of curve names. The client will generate keyshares only for the curve names selected.  \
+                         `none` represents a list of empty keyshares. By default, the client will generate keyshares for all supported curves.\n");
```

Comment:
Can we list the curves supported? I know this is just a testing API, but even as a developer I don't want to have to look at the code to figure out my name options :)

Reply:
We could list the curves supported, but not that this is dependent on the security_policy->ecc_preferences. If a user selects a curve supported by s2n but not in the security policy chosen, we throw a `curve not supported` error message.